doc/modules/ensemble.rst

```diff
@@ -440,61 +440,39 @@ Gradient Tree Boosting
+or Gradient Boosted Degression Trees (GBDT) is a generalization
 of boosting to arbitrary
-differentiable loss functions. GBRT is an accurate and effective
+differentiable loss functions. GBDT is an accurate and effective
 off-the-shelf procedure that can be used for both regression and
-classification problems.  Gradient Tree Boosting models are used in a
+classification problems in a
 variety of areas including Web search ranking and ecology.
 
-The advantages of GBRT are:
-
-  + Natural handling of data of mixed type (= heterogeneous features)
-
-  + Predictive power
-
-  + Robustness to outliers in output space (via robust loss functions)
-
-The disadvantages of GBRT are:
-
-  + Scalability, due to the sequential nature of boosting it can
-    hardly be parallelized.
-
 The module :mod:`sklearn.ensemble` provides methods
-for both classification and regression via gradient boosted regression
+for both classification and regression via gradient boosted decision
 trees.
 
-
 .. note::
 
-  Scikit-learn 0.21 introduces two new experimental implementation of
+  Scikit-learn 0.21 introduces two new experimental implementations of
   gradient boosting trees, namely :class:`HistGradientBoostingClassifier`
   and :class:`HistGradientBoostingRegressor`, inspired by
-  `LightGBM <https://github.com/Microsoft/LightGBM>`_. These fast estimators
-  first bin the input samples ``X`` into integer-valued bins (typically 256
-  bins) which tremendously reduces the number of splitting points to
-  consider, and allow the algorithm to leverage integer-based data
-  structures (histograms) instead of relying on sorted continuous values.
-
-  The new histogram-based estimators can be orders of magnitude faster than
-  their continuous counterparts when the number of samples is larger than
-  tens of thousands of samples. The API of these new estimators is slightly
-  different, and some of the features from :class:`GradientBoostingClassifier`
-  and :class:`GradientBoostingRegressor` are not yet supported.
-
-  These new estimators are still **experimental** for now: their predictions
-  and their API might change without any deprecation cycle. To use them, you
-  need to explicitly import ``enable_hist_gradient_boosting``::
-
-    >>> # explicitly require this experimental feature
-    >>> from sklearn.experimental import enable_hist_gradient_boosting  # noqa
-    >>> # now you can import normally from ensemble
-    >>> from sklearn.ensemble import HistGradientBoostingClassifier
+  `LightGBM <https://github.com/Microsoft/LightGBM>`_.
+
+  These new histogram-based estimators can be **orders of magnitude faster**
+  than :class:`GradientBoostingClassifier` and
+  :class:`GradientBoostingRegressor` when the number of samples is larger
+  than tens of thousands of samples.
+
+  They also have built-in support for missing values, which avoids the need
+  for an imputer. Support for categorical features is also part of the
+  roadmap.
```

Comment:
I would not put "support of categorical features" in the user manual if it's not yet implemented.

Reply:
I thought it'd be a way to "engage" users?
We're not committing to any deadline so it's pretty safe. Open to removing though.

Reply:
I think it's fine to talk about planned features in a blog post or a roadmap but not in a user manual. Otherwise if we documented each feature for which there is an open PR, the user manual would be very frustrating to read.

But let's see what other reviewers think.

Reply:
Yeah the categorical will take more time, I'd leave it out. I'll put down the discussions we had with @ogrisel about it in an issue, and we can go forward from there.